adminservice/src/intTest/groovy/nu/yona/server/InitialGoalCreationTest.groovy

```diff
@@ -1,6 +1,7 @@
 package nu.yona.server
 
 import groovyx.net.http.RESTClient
+import groovyx.net.http.HttpResponseException
```

Comment:
This test might go away. The initialization work is done through your new code. The testing work needs to happen against the /goals/ service in the appservice. The update feature will disappear, as we now have the file to load the goals.